app/views/administrate/application/_sidebar.html.erb

```diff
@@ -11,7 +11,7 @@ as defined by the DashboardManifest.
   <% DashboardManifest::DASHBOARDS.each do |resource| %>
     <li>
       <%= link_to(
-        resource.to_s.titleize,
+        display_resource_name(resource),
```

Comment:
I honestly think that the mapping from a resource name to a model class shouldn't be inferred and should always be explicit. This would obviate the need to do `classify` and `constantize` above. I realize that doesn't have anything to do with this PR, but it would simplify the above logic at least.